colibre/description.html

```diff
@@ -99,8 +99,19 @@ <h4>Stellar Feedback</h4>
         <td>\(v_{\rm kinetic}\)</td>
         <td>{{ "%.4g" % (data.metadata.parameters["COLIBREFeedback:SNII_delta_v_km_p_s"] | float) }} km s\(^{-1}\)</td>
     </tr>
+     <tr>
+        <td>\(\Delta T\)</td>
+        <td>{{ data.metadata.parameters["COLIBREFeedback:SNII_delta_T_K"] | float | format_number }} K</td>
+    </tr>
 </table>
 
+<h4>Gas Cooling</h4>
```

Comment:
Not 100% happy with this - what if somebody uses the old Wiersma tables? Is there not something in SWIFT that actually tells us this?

Reply:
Personally, I'd be very surprised if we decide to use something different alongside the COLIBRE pipeline (in addition to the mentioned two cooling models). 

Nonetheless, I can change `Table cooling (Ploeckinger, 2020)` -> `Table cooling` to make it more general, if you think that'd be better.

Reply:
Ah my point was more that surely there's something in the SWIFT output that tells us what cooling model we're using? There is for hydro, gravity, etc.

Reply:
I am not sure if there is anything in the hydro part of a snapshot that would tell us something about the cooling.

The easiest thing that comes to my mind is to look for a whole block of paramteres `COLIBRECooling:parameters` and `CHIMESCooling:parameters` .

In order to cover all possible cases, my suggestion would be to

- look for something from `COLIBRECooling:parameters`; if we find anything then we are using COLIBRE cooling,
- if not, then look for something from `CHIMESCooling:parameters`; if we get something then we know that CHIMES cooling was adopted
- Say `Unknown`

Reply:
Right but we should probably change this in SWIFT then. We have sections in `SubgridScheme` that should tell us about this.

Reply:
So I've just checked and

`data.metadata.subgrid_scheme["Cooling Model"].decode("utf-8")`

Should give us the cooling model. If it does not for COLIBRE, we need to fix it.

Reply:
Indeed! It works
![Screenshot from 2020-10-09 12-01-21](https://user-images.githubusercontent.com/20153933/95570819-c03a9880-0a27-11eb-97b5-c6046ca540be.png)